projects/demo-playwright/tests/addon-table/table-pagination/table-pagination.spec.ts

```diff
@@ -0,0 +1,73 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, Locator, Page, test} from '@playwright/test';
+
+test.describe(`TablePagination`, () => {
+    test.describe(`Dropdown with \`[size]\`-options (amount items per page)`, () => {
+        test(`Basic case`, async ({page}) => {
+            await tuiGoto(page, `/components/table-pagination/API`);
+            const {apiPageExample} = new TuiDocumentationPagePO(page);
+
+            const {tablePagination, linesPerPageSelect} =
+                await initializeAliases(apiPageExample);
+
+            await linesPerPageSelect.click();
+
+            await waitForCheckmarkIcon(page);
+            await expect(tablePagination).toHaveScreenshot(`0-[size]-dropdown-base.png`);
```

Comment:
Let's decrease screenshot area a little bit
![0--size--dropdown-base-chromium-darwin](https://github.com/taiga-family/taiga-ui/assets/35179038/9cef5105-c149-49e2-a89f-7a17f36d3fae)

Reply:
how small should i go?

![0--size--dropdown-base-chromium-darwin](https://github.com/taiga-family/taiga-ui/assets/29514264/6fdf49ac-8c6b-43d2-b069-d42df39524e8)

Reply:
decreased in 7edb545c0c79854fe9b037e5fd74e91c144444f8